skbio/sequence/_sequence.py

```diff
@@ -373,6 +376,7 @@ def metadata(self):
         return self._metadata
 
     @metadata.setter
+    @stable(as_of="0.4.0")
```

Comment:
Ok, I made that change. It looks like this is only in one place in the code base right now - am I missing any?